anthos-bm-gcp-terraform/main.tf

```diff
@@ -323,6 +323,16 @@ module "init_hosts" {
   cluster_yaml_path      = local.cluster_yaml_file
 }
 
+module "gke_hub_membership" {
```

Comment:
Does this cause any issues when running the scrip for the first time? When there is no cluster membership to delete?

Also doesn't this mean that if this runs right after the `install_abm` phase, then would't it be deleting the membership created the install abm phase?

Reply:
There isn't a `create_cmd_body `, only a `destroy_cmd_body` so on create it is basically a no-op.  The depends_on is primarily for logically sequencing, rather than a strict requirement.